app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_likes_counter
+
```

Comment:
OPTIONAL -I suggest using the after_save method instead, as it ensures that the update is made only when the data is saved in the database. Please fix this :+1: